src/block.c

```diff
@@ -925,17 +926,17 @@ static int sm_block_io(struct super_block *sb, int rw, u64 blkno,
 	return ret;
 }
 
-int scoutfs_block_read_sm(struct super_block *sb, u64 blkno,
+int scoutfs_block_read_sm(struct block_device *bdev, u64 blkno,
```

Comment:
For exported functions like this, let's keep the initial super block arg.  It's sort of a universal file system context.  It's how we get at counters and is needed as an input for tracing.  Let's just add a bdev argument as well.